libs/platform/core/src/services/page-head/page-head.service.ts

```diff
@@ -0,0 +1,16 @@
+import { ElementAttributes, ElementDefinition } from './page-head.model';
+
+export const PageHeadService = 'oryx.PageHeadService';
+
+export interface PageHeadService {
+  addElements(definitions: ElementDefinition | ElementDefinition[]): void;
+  updateElement(definition: ElementDefinition): void;
+  updateHtmlElement(attrs: ElementAttributes): void;
```

Comment:
do we need both `updateHtmlElement` and `setAttributes`?

we just want to be able to crud meta tags with the service (where "read" is not what we need now, but could be added for completness, the same for "delete") so api should be fairly concise  (+ special case for managing html attributes)

Reply:
IMO we don't . We decidet to have separate method for `html`

Reply:
dropped